src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
```

Comment:
There was an alert saying this using directive wasn't necessary, so I removed it as part of this change